custom-login/src/config.js

```diff
@@ -4,7 +4,7 @@ export default {
   oidc: {
     clientId: CLIENT_ID,
     issuer: ISSUER,
-    redirectUri: 'http://localhost:8080/implicit/callback',
+    redirectUri: 'http://localhost:8080/login/callback',
```

Comment:
what's the default value in the admin console?

Reply:
The default value is `/implicit/callback`, but we're trying to get this changed. There's no reason to say "implicit" in the URL for auth code flow. It causes confusion for developers and makes us look bad, IMO. 

We've been using `/callback` on the developer blog since the beginning of 2020. After discussions with @swiftone and @aarongranick-okta, we decided that `/login/callback` is a better solution. We don't plan to update our blog posts because we're specific about the value and everything works.

Reply:
Relevant JIRA: https://oktainc.atlassian.net/browse/OKTA-324376

Reply:
I understand and agree with `/login/callback`. But it would be less confusing if we roll it out in order, like change the admin console value first, then change the value in samples.

Reply:
I'd agree if I thought the admin console would change this year. ;) 

We've been using `/callback` on the blog w/o issues. I've also updated most of the sample READMEs to provide explicit instructions on how to create the app. For example, see https://github.com/okta/samples-js-angular#angular-sample-applications-for-okta.

Reply:
Related: I created an issue to [update our docs](https://github.com/okta/okta-developer-docs/pull/1433), too.